extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -96,12 +101,29 @@ class AmpLightbox extends AMP.BaseElement {
     });
   }
 
+  closeOnClick(event) {
+    if (event.target.hasAttribute('close')) {
```

Comment:
This is not sufficient because the target could be an element inside of the element with attribute `close`.

E.g. it could be `<b>` element in this example:
```
<button close>Close<b>me</b>.</button>
```

I'm leaning more and more toward doing away with `close` attribute.